hal/sparcv8leon/_traps.S

```diff
@@ -673,6 +673,7 @@ s_wovfl_done:
 	FPU_SAVE
 
 s_fpu_done:
+	mov %sp, %o2 /* cpu_context_t * */
```

Comment:
Can be done in delay slot after `call syscalls_dispatch`: change `nop` to `add %sp, 0x60, %o2`
And please update `syscalls_dispatch` signature in the comment in code